app/processors/email_processor.rb

```diff
@@ -1,12 +1,21 @@
 class EmailProcessor
-  cattr_accessor :default_to_newest_feed, default: false
+  def self.for_payload(payload)
+    json = JSON.parse(payload)
+    params = Griddler::Postmark::Adapter.normalize_params(json.deep_symbolize_keys)
+    email = Griddler::Email.new(params)
+    new(email)
+  end
 
-  def initialize(email)
+  def initialize(email, feed: nil)
```

Comment:
I think this `, feed: nil` isn't needed by this PR anymore?

Reply:
correct, thank you!